src/serverroutes.js

```diff
@@ -409,6 +410,17 @@ module.exports = function(app, saveSecurityConfig, getSecurityConfig) {
     res.json(settings)
   })
 
+  app.get(`${SERVERROUTESPREFIX}/baseDeltas`, (req, res, next) => {
+    basedeltas = config.readBaseDeltas(app)
+    res.json(basedeltas)
+  })
+
+  app.put(`${SERVERROUTESPREFIX}/writeBaseDeltasFile`, (req, res, next) => {
```

Comment:
isnt' this PUTting basedeltas, no need to say _write_ and _file_?

Reply:
agreed and changed